admin.py

```diff
@@ -15,8 +15,9 @@
 
 # Add model views
 admin.add_view(RestrictedModelView(Role, db.session, category='Admin'))
-admin.add_view(UserModelView(User, db.session, category='Admin'))
-admin.add_view(BankAdminModelView(Bank, db.session, category='Admin'))
+admin.add_view(AdminUserModelView(User, db.session, category='Admin', endpoint='user_admin'))
```

Comment:
change endpoints to 'user_admin' and 'user_finance'

Reply:
user_admin to admin
and the other to user_finance